src/unread/UnreadCards.js

```diff
@@ -59,6 +61,14 @@ export default function UnreadCards(props: Props): Node {
     ...unreadStreamsAndTopics,
   ];
 
+  const [collapsedStreamIds, setCollapsedStreamIds] = useState(new Immutable.Set<number>());
```

Comment:
`Immutable` containers are initialized with a plain function call, without `new`. See `git grep Immutable` for examples in our codebase.